package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rok8s-scripts",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
This isn't actually a correct version anymore. The versioning happens in the release process. Should we update it to match that?